app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :delete_all
   has_many :likes, dependent: :delete_all
 
+  validates :text, presence: true, length: { maximum: 250, minimum: 1 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   after_create :update_posts_count
```

Comment:
- [ ] According to the project requirements you should add the following validations for the Post model:
- Title must not be blank.
- Title must not exceed 250 characters.
Kindly add this ✍🏻

Reply:
@BenMukebo , this has now been implemented